src/Command.js

```diff
@@ -115,6 +120,9 @@ export default class Command {
       this._execOpts = {
         cwd: this.repository.rootPath,
       };
+      if (this.options.maxBuffer) {
```

Comment:
Minor nit, but I wouldn't mind a newline above this inner conditional block, it helps readability